WhaleSpotting/Models/Response/UserResponse.cs

```diff
@@ -6,6 +6,9 @@ public class UserResponse
 {
     public int Id { get; set; }
     public string Username { get; set; }
+    public string Password { get; set; }
```

Comment:
I don't think we need to return the password in this model as we have methods to check password is correct but we might want to add the `UserType` so we can check if a user is admin and also return the `List<WhaleSighting>` property.